### ISSUE TITLE
Mi Band 1?

### ISSUE THREAD

**Issue by kswt** [creotiv/MiBand2#22]
Is Mi Band 1 supported?

**Comment by creotiv** [comment]
Nope, but there was another lib for MiBand 1 somewhere on internet

Kindly yours,
Andrey Nikishaev

Areas         ML/DS/CV/Soft Dev/BizDev/Growth Hacking/Customer Rel/IT
LinkedIn      http://ua.linkedin.com/in/creotiv
GitHub        http://github.com/creotiv
Slideshare  https://www.slideshare.net/anikishaev/
Skype         creotiv.in.ua
Mobile        +380632410666
